packages/documentation/src/shared/package-tag.tsx

```diff
@@ -0,0 +1,23 @@
+import { MetaComponent } from '@root/types';
+
+type PackageTagProps = {
+  meta: MetaComponent;
+};
+
+export default function PackageTag({ meta }: PackageTagProps) {
+  let packageName = '';
+  let tagColor = '';
+
+  if (meta.tags.includes('package:Styles')) {
+    packageName = '@swisspost/design-system-styles';
+  } else if (meta.tags.includes('package:WebComponents')) {
+    packageName = '@swisspost/design-system-components';
+    tagColor = 'yellow';
+  }
+
+  return (
+    <div className={`tag tag-sm mb-24 tag-${tagColor}`}>
```

Comment:
With the styles package `tagColor` is an empty string, therefore here you are adding an unnecessary `tag-` class.